.build/docker/run-tests.sh

```diff
@@ -213,8 +243,10 @@ docker exec --user root ${container_name} bash -c "\${CASSANDRA_DIR}/.build/dock
 docker exec --user root ${container_name} update-alternatives --set python /usr/bin/python${python_version} | tee -a ${logfile}
```

Comment:
Shouldn't it also be pipefail?

Reply:
In case of fail we just quit with exit status 1, right?

Reply:
this should just exit yes.   it is not expected to ever fail.